build.proj

```diff
@@ -43,7 +43,7 @@
     <Message Importance="High" Text="Restoring all projects..." />
     <MSBuild Projects="@(Project)"
              Targets="Restore" 
-             Properties="MSBuildWarningsAsMessages=NU1503" />
+             Properties="MSBuildWarningsAsMessages=NU1503;RestoreDisableParallel=true" />
```

Comment:
Are we seeing connection issues due to parallel connections on all platforms or only on non-windows platforms? Do we want to consider disabling this only on specific platforms?

Reply:
this issue is observed on windows as well and disable parallel has minor impact on build performance

so I want to change this to see if the download issue is indeed on insufficient connections

---
In reply to: [375017713](https://github.com/dotnet/machinelearning/pull/4776#discussion_r375017713) [](ancestors = 375017713)

Reply:
> this issue is observed on windows as well

The issue on Windows is not likely to be caused by parallel connections. I'm concerned that this change confuses the issue, and leaves the repository in an inferior state for unclear reasons.

/cc @jaredpar

Reply:
I'm not sure about whether parallel connections cause issue on windows and best way to verify is to merge this to master and observe the result, we can't tell the difference from PR build as this issue occurs quite rare.
And disable parallelism has minor impact to build performance so I would like say we give it a try.

---
In reply to: [375543129](https://github.com/dotnet/machinelearning/pull/4776#discussion_r375543129) [](ancestors = 375543129)

Reply:
@frank-dong-ms: @sharwell has a valid concern. But I see your point that we should at least give it a try. Can you please create a P1 bug to track this issue to try and revert it back for Windows at a later date if this PR fixes the other issues?   


---
In reply to: [375566985](https://github.com/dotnet/machinelearning/pull/4776#discussion_r375566985) [](ancestors = 375566985,375543129)

Reply:
Sure, issue below created:
[https://github.com/dotnet/machinelearning/issues/4788](https://github.com/dotnet/machinelearning/issues/4788) 

---
In reply to: [375622405](https://github.com/dotnet/machinelearning/pull/4776#discussion_r375622405) [](ancestors = 375622405,375566985,375543129)